WTWSMS/events/religious_events.txt

```diff
@@ -3763,11 +3809,16 @@ character_event = {
 			
 			# Pragmatism
 			modifier = {
-				factor = 3  # Powerful rulers want a modern religion
+				factor = 4  # Powerful rulers want a modern religion
```

Comment:
If "powerful rulers want a modern religion", then the likelihood to convert should be higher.